commit fc8b9ac29c03a7603a35f393713d1441046a6ccc (refs/changes/86/3586/2)
Author: Hans Rosenfeld <hans.rosenfeld@joyent.com>
Date:   2018-03-12T18:34:25+00:00 (1 year, 7 months ago)
    
    OS-6678 need bhyve modules in name_to_major
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/overlay/generic/etc/name_to_major b/overlay/generic/etc/name_to_major
index 7f49def4..cfab8bff 100644
--- a/overlay/generic/etc/name_to_major
+++ b/overlay/generic/etc/name_to_major
@@ -298,3 +298,6 @@ iwn 301
 xhci 302
 qede 303
 buildstamp 304
+vmm 305
+viona 306
+ppt 307
